packages/delegation-process/aws.config.local

```diff
@@ -0,0 +1,4 @@
+[default]
+aws_access_key_id=testawskey
+aws_secret_access_key=testawssecret
+region=eu-central-1
```

Comment:
See #1128 
```suggestion
region=eu-south-1
```

Reply:
already handled in this separate [PR](https://github.com/pagopa/interop-be-monorepo/pull/1159) that should be merged soon